docs/demos/credit-card-fraud.md

```diff
@@ -0,0 +1,117 @@
+
+- Go to the OpenShift Console as a Developer
+- Select your mlflow project 
+- Press Topology 
+- Press the mlflow-server circle 
+    - While you are at it, you can also press the little "Open URL" button in the top right corner of the circle to open up the MLFlow UI in a new tab - we will need it later.
+- Go to the Resources tab 
+- Press mlflow-server under Services 
+- Look at the Hostname and mlflow-server Port.  
+NOTE: This route and port only work internally in the cluster.
+![Find mlflow-server-service](img/mlflow-server-service.png)
+![Find the hostname and port](img/hostname-and-port.png)
+
+### 1.2: MLFlow Route through oc
+Alternatively, you can use the OC command to get the route through: `oc get route mlflow | grep mlflow`
+
+### 2: Create a RHODS workbench
+Open up RHODS and create a new Data Science project (see image), this is where we will build and train our model. This will also create a namespace in OpenShift which is where we will be running our application after the model is done.  
+I'm calling my project 'Credit Fraud', feel free to call yours something different but be aware that some things further down in the demo may change.
+![Create Data Science Project](img/Create_data_science_project.png)
+
+After the project has been created, create a workbench where we can run Jupyter.  
+There are a few important settings here that we need to set:
+
+- **Name:** I simply call it "Credit Fraud Model", but feel free to call it something else.
+- **Notebook Image:** Standard Data Science
+- **Deployment Size:** Small
+- **Environment Variable:** Add a new one that's a Config Map -> Key/value and enter `MLFLOW_ROUTE` as the key and `http://<route-to-mlflow>:<port>` as the value.  
+This is the same route and port that we found in [step one](#11-mlflow-route-through-visual-interface), in my case it is `http://mlflow-server.mlflow.svc.cluster.local:8080`.
+- **Cluster Storage:** Create new persistent storage - I call it "Credit Fraud Storage" and set the size to 20GB.
+
+Press Create Workbench and wait for it to start - status should say "Running" and you should be able to press the Open link
+![Workbench Settings](img/Workbench_Settings.png)
+![Workbench](img/Workbench.png)
+
+Open the workbench and login if needed.
+
+### 3: Train the model
+When inside the workbench (Jupyter), pull down this repo from GitHub: [https://github.com/red-hat-data-services/credit-fraud-detection-demo](https://github.com/red-hat-data-services/credit-fraud-detection-demo).   
+It contains:
+
+- A notebook (model.ipynb) inside the `model` folder with a Deep Neural Network model we will train.
+- Data for that model.
+- An application (model_application.py) inside the `application` folder that will fetch the trained model from MLFlow and run a prediction on it whenever it gets any user input.
+![Jupyter](img/Jupyter.png)
+
+Take a look inside the folders and scripts if you are interested.  
```

Comment:
You said at the beginning that this demo would show people how to train and track models. So "if you are interested" seems odd. If it's optional you lose half of the purpose of the demo.

Reply:
Very good point 😅 
I added a walkthrough of the parts of the code that are relevant to interacting with MLFlow and training the model, let me know if that flows better.